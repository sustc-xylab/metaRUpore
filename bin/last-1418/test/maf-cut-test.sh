#! /bin/sh

cd $(dirname $0)

PATH=../bin:$PATH

{
    maf-cut chr1:152413-158286 split1.maf

    maf-cut chrM:150-200 bs100.maf bs100.fastq

    awk '(NR-1) % 4 < 2' bs100.fastq | tr '@' '>' |
	maf-cut chrM:150-200 bs100.maf -
} | diff -u maf-cut-test.out -
