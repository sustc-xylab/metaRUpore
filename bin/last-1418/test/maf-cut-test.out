a score=192 mismap=0.117
s KI635862   2418195 86 +  12728579 CACAGGCCGCTGCAGCCCAGAACTCCTGGCCTCAAGCGATCCACCAGCCTCAGCCTC-CGAGCAGCCGG-ATAACAGGCGCGCGCCAC
s chr1     248803921 88 - 248956422 CATAGCTCACTGCAGCCTTGACCTCCTAGGCTAAAGCAATCCTCCCACCTTAGCCTCTCCAGTAGCTGGAACTACAGGCATGCATCAC
p                                   #$$%&&'(()******************************************************************************

a score=330 mismap=0.75
s KI636063 409336 105 +    789923 AAAATATGGAACGCTTCACGAATTTGCGTGTCATCCTTGCGCAGGGGCCATGCTAATCTTCTCTGTATCGTTCCAATTTTAGTATATGTGCTGCCGAAGCGAGCA
s chr1     157784 102 + 248956422 aaaataTGGAATGCTTCACAAATTTGCATGTCATTCTTTCACAGAGGCCGTGCCAA---TCTCTCTATTGTTCCAACTTAAGTATGTGTGCTACTGAGGCAAGCA
p                                 !"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""!

a score=203 mismap=0.0127
s KI635862   2418239 48 +  12728579 CAGCCTCAGCCTC-CGAGCAGCC-GGATAACAGGCGCGCGCCACCGCGCC
s chr1     248798136 50 - 248956422 CTGCCTCAGCCTCACAAGTAGCTGGGACTACAGGTGCTTGTCACCACACC
p                                   33333333333333333333333333333333333322211110.*)'&$

@ 7 87 chrMa 141 -
TAGTAGGTATGTTTGCTTGTAATATTGAATGTAGGAGCGATAGATAAGAG
+
?:+4>89ABBB@AB?):B=5=1@@49;;9;<9(22+/+>?060.=<4++2
@ 21 87 chrMa 80 +
TTATTATTTATCGTATT
+
B2BBABBA:1:A8A?=9
@ 95 87 chrMa 131 +
TTATTATTTATCGTATTTACGTTTAATATTATAGGTGAATATATTTATTG
+
A?6BA=*?@>>8:?0;=6>@A8?76@7<?@?;99174=9?3216;(:19#
> 7 87 chrMa 141 -
TAGTAGGTATGTTTGCTTGTAATATTGAATGTAGGAGCGATAGATAAGAG
> 21 87 chrMa 80 +
TTATTATTTATCGTATT
> 95 87 chrMa 131 +
TTATTATTTATCGTATTTACGTTTAATATTATAGGTGAATATATTTATTG
